docs/_sources/code/tests.rst.txt

```diff
@@ -0,0 +1,215 @@
+Tests
+-----
+
+This section contains the description of the functional UI test created with the selenium testframework. The Pagination
```

Comment:
*This section contains the description of the functional UI tests created with the Selenium testing framework.